app/views/layouts/application.html.erb

```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title><%= t('title_tag') %></title>
```

Comment:
Nice! Now to could DRY it up with a little helper, like

```
module ApplicationHelper
  def page_tag(tag_name)
    t("#{controller_name}.#{action_name}.#{tag_name}")
  end
end
```

and then in the layout

`<title><%= page_tag('title_tag') %></title>`

or something :)